### ISSUE TITLE
[macOS] Elements accessibility bug

### ISSUE THREAD

**Issue by leszaj** [unoplatform/uno#8047]
### Current behavior

1. I have the multiplatform application developped using UNO Platform.

2. I would like to test using appium. 

3. Application is fully fucntional when it comes to manual usage.

**Win behaviour:**

4. Accesibility inspector is able to inspect all of the application content

**MacOs behaviour:**

5. Accesibility inspector can inspect default MacOS window buttons like "close application", "maximalize", and "minimalize" 

6. Accesibility inspector cannot inspect almost all of the application connetnt cannot be accessed(they are not listed at all) as a children of the main window. Only images are accessible but none of the buttons neither texts.

7. I inspected other applications on MacOs using same accesibility inspector (calculator, visual studio, etc.) - all of the elements were accessible.

### Expected behavior

7. Accessibitlity inspector shall be able to inspect all of the visible elemenets.

### How to reproduce it (as minimally and precisely as possible)

1. Clone and build the following repo:
https://github.com/unoplatform/Uno.Ch9

2. Run any accessibilitity instpector program.

3. Inspect application tree - some most of visible elements is missing
4. Try to investigate for instance button elements or texts - not found any of them

### Anything else we need to know?

Issue was described on the stack overflow under the following link as well:
https://stackoverflow.com/questions/70769853/uno-application-run-on-macos-problem-with-elements-accessibility

**Comment by MartinZikmund** [comment]
@ajpinedam Andres, could you check what would be the required effort on this one please?

**Comment by MartinZikmund** [comment]
This is no longer relevant as we have a new macOS backend 🚀
